src/expansion_context.mli

```diff
@@ -28,12 +52,15 @@ module Deriver : sig
   (** Return the code path for the given context *)
   val code_path : t -> Code_path.t
 
+  (** Return the ocaml-migrate-parsetree configuration for the given expansion context *)
+  val omp_config : t -> Migrate_parsetree.Driver.config
+
   (** Wrap a [fun ~loc ~path] into a [fun ~ctxt] *)
   val with_loc_and_path : (loc:Location.t -> path:string -> 'a) -> (ctxt:t -> 'a)
 
   (**/**)
   (** Undocumented section *)
 
   (** Build a new expansion context with the given item location and code path *)
-  val make : derived_item_loc:Location.t -> code_path:Code_path.t -> t
+  val make : derived_item_loc:Location.t -> base:Base.t -> t
```

Comment:
If we want to make this a bit more future proof, we could add a unit argument. This way, we could add optional arguments to this function in the future.

Reply:
Good point, I'll add that!